dail/postponed-expedited-snap-verifications.vbs

```diff
@@ -104,13 +201,45 @@ DO
 		Dialog Dialog1
 		cancel_confirmation
 		Call validate_MAXIS_case_number(err_msg, "*")
-		IF verifs_needed = "" THEN err_msg = err_msg & vbCr & "* Please enter the postponed verifications requested."
-		IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
+		IF trim(verifs_needed) = "" THEN err_msg = err_msg & vbCr & "* Please enter the postponed verifications requested."
+		IF trim(worker_signature) = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
 		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine		'error message including instruction on what needs to be fixed from each mandatory field if incorrect
 	LOOP UNTIL err_msg = ""									'loops until all errors are resolved
 	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
 
+'Navigates to SPEC/MEMO 
+Call write_value_and_transmit("P", 6, 3)
+Call write_value_and_transmit("MEMO", 20, 70)
+'Function to create new MEMO
+Call start_a_new_spec_memo(memo_opened, False, forms_to_arep, forms_to_swkr, send_to_other, other_name, other_street, other_city, other_state, other_zip, False)
```

Comment:
The arguments at 'forms_to_arep' and 'forms_to_swkr' should be 'Y' and 'send_to_other' as 'N' to have the function send the MEMO to an Authorized Representative or Social Worker if they exists. Send to other allows us to send a MEMO to any address, which is not likely helpful in this situation. 

I would recommend the last argument be 'True' this will have the script stop if a MEMO is not started correctly.

Reply:
Thanks for discussing. I updated the function accordingly.